scripts/deploysnapshot.sh

```diff
@@ -21,6 +21,7 @@ SNAPSHOTDIR=/var/www/vhosts/mf-geoadmin3/private/snapshots/$1
 # Prepare snapshot
 cd $SNAPSHOTDIR/geoadmin/code/geoadmin
 if [ "$NEW_VERSION" = "true" ]; then
+  export KEEP_VERSION='false'
```

Comment:
No per default KEEP_VERSION is 'true' in Makefile for rc_(dev|int|prod) it will be re-applied later when sourcing those files.